.github/workflows/cd.yml

```diff
@@ -77,7 +88,6 @@ jobs:
         working-directory: deluge_src
         run: |
           python -m pip install .
-          python setup.py install_scripts
```

Comment:
Are you sure removing this is correct?

Reply:
Nope. I think it might be with something I did on another branch but I'll cross that bridge when I get there.
Reverted for now.